sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -416,7 +416,7 @@ public static RuleFilter GetFilter(AmqpRuleFilterCodec amqpFilter)
                         MessageId = amqpCorrelationFilter.MessageId,
                         To = amqpCorrelationFilter.To,
                         ReplyTo = amqpCorrelationFilter.ReplyTo,
-                        Label = amqpCorrelationFilter.Label,
+                        Subject = amqpCorrelationFilter.Label,
```

Comment:
nit:  Should we consider renaming the filter member name too?  _(disclaimer: I did not dig in to see if this is owned by us or defined by the AMQP lib)_

Reply:
Sure, it is an internal type owned by us.